### PR TITLE
fix a UI bug on firefox

### PR DESCRIPTION
**Pull request by Renewwen**

### CODE REVIEW

File: bnbridge/src/App.js [hunk 1]
```diff
@@ -21,18 +21,19 @@ function App() {
       <Grid
         style={{
           padding: '5% 10% 5% 10%',
-          width: '50%',
+          //width: '50%',
```
Comment (denniswon) [inline bnbridge/src/App.js]: remove this

File: bnbridge/src/App.js [hunk 2]
```diff
@@ -21,18 +21,19 @@ function App() {
...
           minWidth: 'fit-content',
-          marginLeft: '15%',
+          //marginLeft: '15%',
```
Comment (denniswon) [inline bnbridge/src/App.js]: remove this

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (denniswon) [inline package-lock.json]: delete this file

**Review by denniswon — CHANGES_REQUESTED** [review]
update with description and test plan for screenshot for the update/fix.

**Comment by Renewwen** [comment]
Fixed the UI display bug on Firefox browser: 
The Previous:
image
After fixed:
image

**Comment by denniswon** [comment]
lgtm!

**Pull request merged**